<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../recursos/login_rc.qrc">
    <normaloff>:/Icono/images/icono.ico</normaloff>
    <selectedon>:/Icono/images/logo.ico</selectedon>:/Icono/images/icono.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_error">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #dc95c4;
border-radius: 5px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_error">
            <property name="styleSheet">
             <string notr="true">color: rgb(35, 35, 35);</string>
            </property>
            <property name="text">
             <string>Error</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_close_pupup">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border-radius: 5px;	
	background-image: url(:/Salir/images/cil-x.png);
	background-position: center;	
	background-color: rgb(60, 60, 60);
}
QPushButton:hover {
	background-color: rgb(50, 50, 50);	
	color: rgb(200, 200, 200);
}
QPushButton:pressed {
	background-color: rgb(35, 35, 35);	
	color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="content">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Fondo/images/fondo.jpg);
background-repeat: no-repeat;
background-position: center;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="login_area">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>441</width>
         <height>541</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>550</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
border-radius: 10px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="logo">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>0</y>
          <width>361</width>
          <height>311</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Logo/images/logo.png);
background-repeat: no-repeat;
background-position: center;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_user">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>327</y>
          <width>280</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial,sans-serif</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #e4acd0; /* Borde */
    border-radius: 5px;
    padding: 5px;
    background-color: #f5eef2; /* Fondo */
    color: #dc84bc; /* Color del texto */
    font-family: Arial, sans-serif;
    font-size: 14px;
}
QLineEdit:hover {
	border: 2px solid rgb(255, 255, 255);
}
QLineEdit:focus {
    border: 2px solid #dc84bc; /* Borde al enfocar */
}</string>
        </property>
        <property name="maxLength">
         <number>32</number>
        </property>
        <property name="placeholderText">
         <string>USER</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>390</y>
          <width>280</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial,sans-serif</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #e4acd0; /* Borde */
    border-radius: 5px;
    padding: 5px;
    background-color: #f5eef2; /* Fondo */
    color: #dc84bc; /* Color del texto */
    font-family: Arial, sans-serif;
    font-size: 14px;
}
QLineEdit:hover {
	border: 2px solid rgb(255, 255, 255);
}
QLineEdit:focus {
    border: 2px solid #dc84bc; /* Borde al enfocar */
}</string>
        </property>
        <property name="maxLength">
         <number>16</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>PASSWORD</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_login">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>480</y>
          <width>280</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: 2px solid #e4acd0;
    border-radius: 5px;
    padding: 8px 16px;
    background-color: #dc95c4; /* Fondo */
    color: #ffffff; /* Color del texto */
    font-family: Arial, sans-serif;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #dc84bc; /* Fondo al pasar el cursor */
}

QPushButton:pressed {
    background-color: #d484b4; /* Fondo al presionar */
}</string>
        </property>
        <property name="text">
         <string>CONNECT TO CYBERPUNK</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../recursos/login_rc.qrc"/>
 </resources>
 <connections/>
</ui>
